code/modules/power/port_gen.dm

```diff
@@ -315,13 +315,3 @@ display round(lastgen) and plasmatank amount
 	board_path = "/obj/item/weapon/circuitboard/pacman/super"
 	overheat()
 		explosion(src.loc, 3, 3, 3, -1)
-
-/obj/machinery/power/port_gen/pacman/mrs

```

Comment:
Mrs. Pacman is definitely a thing on TG--the crazy thing eats DIAMONDS for power.